.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,41 @@
+name: unit-tests
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
please copy all the other qualifiers so that we dont run tests on for example on README changes/etc

https://github.com/crashappsec/chalk/blob/b583d27c6648b6860149b3b3550a29185c9015ed/.github/workflows/tests.yml#L7-L13

Reply:
path ignore is not equivalnt to path opt-in.

for example this will trigger tests when updating `.clabot` file which is not related to tests

please switch to use existing pattern for consistency